README.md

```diff
@@ -75,6 +75,17 @@ img_info = FormatParser.parse(File.open("myimage.jpg", "rb"))
 JSON.pretty_generate(img_info) #=> ...
 ```
 
+To convert the result to a Hash
+
+```ruby
+img_info = FormatParser.parse(File.open("myimage.jpg", "rb"))
+img_info.as_json
+
+# it's also possible to convert all keys to string (since version 0.22)
```

Comment:
The README is attached to a specific version of the gem already, so maybe not necessary to mention in which version a certain feature has become available. If it is in the README it means you have it :-)

Reply:
Yeah, I was not sure about adding this information. I will remove it!